src/Select/Select.test.js

```diff
@@ -51,7 +51,7 @@ describe('<Select />', () => {
         });
     });
 
-    test('forwards the ref to the button', () => {
+    xtest('forwards the ref to the div role="button"', () => {
```

Comment:
@skvale I've done something wrong with how to handle forwarding the ref in Select - could you take a look when you get some time?

Reply:
Popover is overriding the ref prop of this control element
```js
        let controlProps = {
            onClick: onClickFunctions,
            ref: (c) => {
                this.controlRef = findDOMNode(c);
            }
        };
```
I'll see if there's way to extend the ref, but still maintain the original ref somehow

Reply:
I think we need to do something like this in Popover
```js
let controlProps = {
    onClick: onClickFunctions,
    ref: (c) => {
        this.controlRef = findDOMNode(c);
        if (typeof control.ref === 'function') {
            control.ref(c);
        } else if (typeof control.ref === 'object') {
            control.ref.current = c;
        }
    }
};
```